lib/options.json

```diff
@@ -296,8 +296,8 @@
     "public": {
       "type": "string"
     },
-    "requestCert": {
-      "type": "boolean"
+    "publicPath": {
```

Comment:
why does this pr have publicPath changed?

Reply:
@hiroppy merge problem, fixed